src/main/webapp/app/modules/data-status/data-status.tsx

```diff
@@ -0,0 +1,130 @@
+import './data-status.scss';
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { Row, Col, Card, CardBody, Table } from 'reactstrap';
+import { getPaginationItemsNumber, getSortState, IPaginationBaseState, JhiPagination, TextFormat, Translate } from 'react-jhipster';
+import { fetchDataStatus } from './data-status.reducer';
+import { APP_DATE_FORMAT } from 'app/config/constants';
+import PageSizeSelector from 'app/entities/page-size-selector';
+import { FIRST_PAGE, MAX_BUTTONS } from 'app/shared/util/pagination.constants';
+
+export interface IDataStatusProp extends StateProps, DispatchProps {}
+
+export interface IDataStatusState extends IPaginationBaseState {
+  dropdownOpenTop: boolean;
+  dropdownOpenBottom: boolean;
```

Comment:
where are we using this property? I believe we can remove it and rename 'dropdownOpenTop' to 'dropdownOpen'